packages/core/src/render/api_flags.ts

```diff
@@ -41,7 +41,7 @@ export interface RendererType2 {
   data: {[kind: string]: any};
 
   /**
-   * A function added by the {@link ɵɵExternalStylesFeature} and used by the framework to create
+   * A function added by the [ɵɵExternalStylesFeature](/api/core/ɵɵExternalStylesFeature) and used by the framework to create
```

Comment:
```suggestion
   * A function added by the [ɵɵExternalStylesFeature](/api/core/ɵɵExternalStylesFeature) and used by the framework to create
```

We should drop the link here, symbols prefixed by `ɵ` are not part of the public API and shouldn't be linked.